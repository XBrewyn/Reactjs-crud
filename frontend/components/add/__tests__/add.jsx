import React from 'react';
import { create } from 'react-test-renderer';
import Homepage from '../add';

describe('<Add />', () => {
  it('Renders correctly', () => {
    const component = create(<Homepage />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
