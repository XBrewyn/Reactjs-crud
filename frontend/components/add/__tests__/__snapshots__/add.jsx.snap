// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Add /> Renders correctly 1`] = `
<div
  className="css-4koet0"
>
  <h1>
    Add
  </h1>
  <form>
    <input
      name="name"
      onChange={[Function]}
      placeholder="name"
      type="text"
      value=""
    />
    <input
      name="password"
      onChange={[Function]}
      placeholder="password"
      type="password"
      value=""
    />
    <input
      onClick={[Function]}
      type="button"
      value="add"
    />
  </form>
</div>
`;
